src/i18n/en.json

```diff
@@ -168,31 +171,38 @@
     "doGoToAccountManagement": "Go to account management",
     "authenticationMethod": "Authentication method",
     "primaryAddress": "Primary address",
-    "permanentAddress": "EN: Vakinainen osoite",
-    "temporaryAddress": "EN: Väliaikainen osoite",
-    "permanentForeignAddress": "EN: Vakinainen ulkomainen osoite",
+    "permanentAddress": "Permanent address",
+    "temporaryAddress": "Temporary address",
+    "permanentForeignAddress": "Permanent foreign address",
+    "primaryEmail": "Primary email",
     "primaryPhone": "Primary phone number",
     "primaryAddressAriaDescription": "This is your primary address",
+    "primaryEmailAriaDescription": "This is your primary email",
     "primaryPhoneAriaDescription": "This is your primary phone number",
-    "verifiedDataInformationPrefix": "EN: Merkityt",
-    "verifiedDataInformationAriaPrefix": "EN: Alla olevat",
-    "verifiedDataInformation": "EN: <0></0> tiedot on vahvistettu virallisesta väestötietojärjestelmästä, eikä niitä ei voi tämän takia muokata Helsinki Profiilissa. Voit tutustua halutessasi väestötietojärjestelmään tallennettuihin tietoihisi suomi.fi verkkopalvelussa: <1>siirry suomi.fi palveluun </1>",
+    "verifiedDataInformationPrefix": "Marked",
+    "verifiedDataInformationAriaPrefix": "Those below",
+    "verifiedDataInformation": "<0></0> the data has been verified from the official Population Information System and they can thus not be edited in the Helsinki Profile. You can check your data in the Population Information System through the suomi.fi web service: <1>go to the suomi.fi service </1>",
     "verifiedDataInformationLink": "https://testipalvelu.apro.tunnistus.fi",
-    "verifiedData": "EN: Vahvistettu tieto",
-    "verifiedBasicData": "EN: Vahvistetut perustiedot",
+    "verifiedData": "Verified data",
+    "verifiedBasicData": "Verified basic data",
     "phones": "Phone numbers",
+    "emails": "Emails",
     "addresses": "Addresses",
     "ariaComboBoxHelperText": "Select {{field}}. Search by typing and select one option. {{requiredInfo}}",
     "ariaInputFieldHelperText": "Write {{field}}.  {{requiredInfo}}",
-    "ariaListTitlePhones": "EN: Antamasi puhelinnumerot",
-    "ariaListTitleAddresses": "EN: Antamasi osoitteet",
+    "ariaListTitlePhones": "Phone numbers that you have provided",
+    "ariaListTitleEmails": "Email adresses that you have provided",
```

Comment:
Typo: "adresses" should be "addresses".

Reply:
Good catch. Fixed in excel + translations.